src/KafkaFlow.Serializer/ConfigurationExtensions.cs

```diff
@@ -1,10 +1,9 @@
-namespace KafkaFlow.Serializer.ApacheAvro
+namespace KafkaFlow.Serializer
```

Comment:
Please see [this comment](https://github.com/Farfetch/kafka-flow/pull/140/files#r615706263).